example/simple-mutation/src/main/java/com/oceanbase/example/SimpleMutation.java

```diff
@@ -121,6 +143,51 @@ public static void main(String[] args) {
                         batchResult.get(idx).getAffectedRows()));
             }
 
+            /*
+             * notice. these are only demo for new operations
+             */
+
+            // aggregation
+            // construct aggregation
+            ObTableAggregation obtableAggregation = client.aggregate("aggregation_table");
+
+            // add aggregation operation
+            obtableAggregation.max("c2");
+
+            // execute aggregation
+            ObTableAggregationResult obtableAggregationResult = obtableAggregation.execute();
+
+            // get aggregation result
+            System.out.println(obtableAggregationResult.get("max(c2)"));
+
+            // auto increment
+            // if rowkey is auto increment
+            // use auto increment value, should fill 0
+            // the value of new row on c1 will be 1
+            client.insert("auto_inc_table", new Object[] { 0 }, new String[] {"c2"},
+                    new Object[] { 1 });
+
+            // assign the specific value on auto increment column
+            // the value of new row on c1 will be 100
+            client.insert("auto_inc_table", new Object[] { 100 }, new String[] {"c2"},
+                    new Object[] { 1 });
+
+            //assign the specific value will refresh the auto increment value
+            // the value of new row on c1 will be 101
+            client.insert("auto_inc_table", new Object[] { 0 }, new String[] {"c2"},
+                    new Object[] { 1 });
+
+            // check and insert
+            // use setRowkey to specify the key of the new row
+            // use addScanRange for query scan
```

Comment:
Better not to mention `query`, just tell the user: to add a rowkey range where to check their filter.

Reply:
fixed